### ISSUE TITLE
plot_cladogram

### ISSUE THREAD

**Issue by Roedani** [yiluheihei/microbiomeMarker#33]
Dear@yiluheihei
I was able to run the lefse function and get the Formal calls microbiomMarker, also I could run the plot_ef_bar function, however when I tried to run the plot_cladogram they showed a error:
plot_cladogram(mm, color = c(Day0 = "blue", Day21 = "red"))
Error in `levels<-`(`*tmp*`, value = as.character(levels)) : 
  factor level [2] is duplicated

Could you please help on it?

Here my phylo:
phyloseq-class experiment-level object
otu_table()   OTU Table:         [ 140 taxa and 40 samples ]
sample_data() Sample Data:       [ 40 samples by 4 sample variables ]
tax_table()   Taxonomy Table:    [ 140 taxa by 7 taxonomic ranks ]

Here my microbiomeMarker:
mm
microbiomeMarker-class inherited from phyloseq-class
normalization method:              [ CPM ]
microbiome marker identity method: [ lefse ]
marker_table() Marker Table:       [ 71 microbiome markers with 5 variables ]
otu_table()    OTU Table:          [ 362 taxa and  40 samples ]
sample_data()  Sample Data:        [ 40 samples by  4 sample variables ]
tax_table()    Taxonomy Table:     [ 362 taxa by 1 taxonomic ranks 

Thanks in advance,
Rodrigo

**Comment by yiluheihei** [comment]
@Roedani  Sorry for the delayed reponse. Thanks for interest in microbiomeMarker. 

Please provide a reproducible example, it will be much easier to solve your problem. https://stackoverflow.com/questions/5963269/how-to-make-a-great-r-reproducible-example.

**Comment by Roedani** [comment]
@yiluheihei I am sorry I missed you response.
Thanks so much for getting back to me. I did something different, instead to use the microbiomeMarker, which looks very helpful pipeline I used the ggtree direct. I will try few more time using the microbiomeMarker and if I wont get there I will write to you one more time. I need to load the data again to give you a reproducible example. 
Thanks so much,
Rodrigo
